src/Button.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "styled-components";
+
+function Button(props) {
+  return (
```

Comment:
Nice job, might want to add some more dynamic behaviour.

You could add props for color, background, border, etc and then pass it to the styled component.

Example:
`props.buttonOptions = { background: 'red', color: 'blue', border: '1px solid grey' }`

```
<StyledButton type={props.type} value={props.value} buttonOptions={props.buttonOptions} >
      {props.content}
</StyledButton>

```

```
const StyledButton = styled.button`
  border: ${(props) => props.buttonOptions.border};
  opacity: 0.8;
  color: ${(props) => props.buttonOptions.color};
  font-size: 24px;
  width: 100%;
  font-family: "Sue Ellen Francisco", cursive;
  background: ${(props) => props.buttonOptions.background};
`;
```

You can also set default props to default to certain things if nothing is inputed. 

https://www.geeksforgeeks.org/reactjs-defaultprops/#:~:text=The%20defaultProps%20is%20a%20React,default%20props%20for%20the%20class.